vm/systemSmartContracts/delegation.go

```diff
@@ -659,6 +699,7 @@ func (d *delegation) unStakeNodes(args *vmcommon.ContractCallInput) vmcommon.Ret
 		return vmcommon.UserError
 	}
 	if vmOutput.ReturnCode != vmcommon.Ok {
+		d.eei.AddReturnMessage(vmOutput.ReturnMessage)
```

Comment:
this duplicates the return message already save from auction - i think

Reply:
This pattern is used also in init method above and I found it also in other places. We should decide which is the correct approach